packages/joy-actors/src/AvailableRoles/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { BareProps } from '@polkadot/ui-app/types';
+import { ComponentProps } from '../props';
+import { Role, RoleParameters } from  '../types';
+import { Option, u32, Balance, BlockNumber } from '@polkadot/types';
+import { withCalls } from '@polkadot/ui-api/index';
+import { Table } from 'semantic-ui-react';
+import Section from '@polkadot/joy-utils/Section';
+
+import BN from 'bn.js';
+
+type Props = BareProps & ComponentProps;
+
+export default class AvailableRoles extends React.PureComponent<Props> {
+    render() {
+        return (
+            <div> {this.props.roles.map((role) => <div key={role.toString()}><RoleDisplay role={role}></RoleDisplay></div>) } </div>
+        )
+    }
+}
+
+
+type RoleProps = BareProps & {
+    role: Role,
+    roleParams?: Option<RoleParameters>
+}
+
+class RoleDisplayInner extends React.PureComponent<RoleProps> {
+    render() {
+        const {role, roleParams} = this.props;
+        if (!roleParams) return null;
+
+        const params = roleParams.unwrapOr(undefined);
+        if (!params) return null; // no role parameters defined for this role
+
+        return (
+            <Section title={`${role.toString()}`}>
```

Comment:
You want `${value}` if there should be something around that value in a string, e.g. `My age is ${age} years old`
```suggestion
            <Section title={role.toString()}>
```